src/items/static-items.md

```diff
@@ -59,50 +59,6 @@ unsafe fn bump_levels_unsafe2() -> u32 {
 Mutable statics have the same restrictions as normal statics, except that the
 type does not have to implement the `Sync` trait.
 
-## `'static` lifetime elision
```

Comment:
Not sure how, but it'd be nice for this page to say it's possible and link to the rules for it.

Reply:
There is at least a mention of this on the constant items page with a link.